packages/common/container-definitions/src/loader.ts

```diff
@@ -549,6 +549,8 @@ export interface ILoader extends Partial<IProvideLoader> {
  * The Host's view of the Loader, used for loading Containers
  * @legacy
  * @alpha
+ * @deprecated Use the standalone apis instead to create detached containers namely {@link @fluidframework/container-loader#rehydrateDetachedContainerFromSnapshot}
```

Comment:
Nit: is the plan to remove this in `2.20.0`? If so, should we note that here?

Reply:
We would need changes in Loop before we remove it. So, we are not sure.
Also, we might not remove it, rather can just make it internal. So, avoiding adding that here for now.

Reply:
Marking something as internal is equivalent to removing it as far as our customers are concerned, so I think leaving such comments are still appropriate in those cases. But if we don't know when we will be removing it from the externally visible API surface, not noting a timeline seems fine.